core/App/types/state.ts

```diff
@@ -11,6 +11,7 @@ export interface Onboarding {
 
 export interface Preferences {
   useBiometry: boolean
+  developerModeEnabled: false
```

Comment:
Should the type be `boolean` or is it expected that this will always be false?

Reply:
Missed this one. Will fix to boolean.